modules/ROOT/pages/contribute.adoc

```diff
@@ -7,15 +7,12 @@ Many of our documentation repositories are open and we welcome your contribution
 
 To contribute to the English version of this documentation:
 
-. Select *Edit in Github* on the page you want to modify.
+. If this is your first time contributing, sign the https://cla.salesforce.com/sign-cla[Salesforce.com Contributor License Agreement (CLA)]. The CLA only needs to be signed once per GitHub account.
+. Click *Edit in Github* on the page you want to modify.
+. Make your edits.
 . Submit a pull request to the Git repository.
-. If this is the first time you are submitting a pull request,
-run the https://api-notebook.anypoint.mulesoft.com/notebooks/#d28e8f95e739ee1e1e30d5ec577f4a27[API Notebook].
-If you aren't already logged in to your GitHub account, you can do so using a link provided in the API Notebook.
 
-image:api-notebook1.png[User interface for running the API notebook to become a contributor]
+If you submit the pull request without signing the CLA first, you will receive an error in the pull request. Sign the CLA to remove this error.
```

Comment:
This seems gratuitous to me, given Step 1 of the task.
If we must include it because Support has found that this is a common occurrence, then maybe more of a reminder, something like the following?
```suggestion
If your pull request results in an error, verify that you completed Step 1.
```
Are other errors likely? If so, we might want to add something like "...Step 1 before asking for help."